mesonbuild/wrap/wrap.py

```diff
@@ -490,6 +506,26 @@ def get_git(self) -> None:
             if push_url:
                 verbose_git(['remote', 'set-url', '--push', 'origin', push_url], self.dirname, check=True)
 
+    def validate(self) -> None:
+        # This check is only for subprojects with wraps.
+        if not self.wrap.has_wrap:
+            return
+
+        # Retrieve original hash, if it exists.
+        hash_of_wrap_file_filename = self.wrap.form_hash_file_path(self.dirname)
```

Comment:
I would use `hashfile`

Reply:
Renamed form `hash_of_wrap_file_filename` to `hashfile`